src/app/views/loan-detail/transaction-history/transaction-history.component.html

```diff
@@ -1,7 +1,9 @@
+<ngx-spinner bdColor="rgba(0,0,0,.3)" size="medium" color="#fff" class="overlay-setup" type="ball-clip-rotate-multiple" id ="spinner" #spinner></ngx-spinner>
 <div class="transaction-component">
   <div class="timeline">
 
     <div class="timeline-structure">
+      <h2 *ngIf="noMatch" class="title main-h1" style="text-transform: initial; text-align: center; font-weight: 700;"> History is not available </h2>
```

Comment:
**Cosider using the same structure that requested loans**

  "<div *ngIf="noMatch" class="title main-h1 text-center">History is not available</div>"